src/app/app.component.spec.ts

```diff
@@ -24,6 +25,7 @@ describe('AppComponent', () => {
         NavbarComponent,
         MainPageComponent,
         PageNotFoundComponent,
+        BooksListComponent,
         MatToolbar
       ],
     }).compileComponents();
```

Comment:
This was just to try remove a warning from a unit test. It didn't actually work, but I think it's convention to remember to put new components in your main unit test module? idk, will need to read up on this